examples/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="eng">
+<link rel="stylesheet" href="styles.css">
```

Comment:
**suggestion:** <link> and <script> tags are usually inside the <head> tag of the html (or inside the <body> tag in some cases)